Code/Framework/AzCore/AzCore/Component/TransformBus.h

```diff
@@ -302,6 +302,13 @@ namespace AZ
 
         //! Set the behavior at runtime when this entity's parent's transform changes.
         virtual void SetOnParentChangedBehavior([[maybe_unused]] OnParentChangedBehavior onParentChangedBehavior) {}
+
+        // carbonated begin aoreshko game_specific_1
```

Comment:
Is "game_specific_1" the branch name?
It is supposed to be an unique git branch name like feature/game_specific1. Game specific can be a comment (justification text), but the branch name should be more meaningful. Naming them game_specific_1, game_specific_2 makes not so much sense.